packages/react/src/components/Tile/Tile.js

```diff
@@ -485,15 +489,24 @@ export class ExpandableTile extends Component {
         : this.state.tileMaxHeight + this.state.tilePadding,
     };
     const content = this.getChildren().map((child, index) => {
-      return React.cloneElement(child, { ref: index });
+      // Changes aria-hidden above/below the fold content based on the state of expanded so the content doesn't get announced by screenreaders at the wrong time.
+      const isHidden = index === 0 ? expanded : !expanded;
+
+      return React.cloneElement(child, {
+        ref: index,
+        'aria-hidden': isHidden,
+      });
     });
-    const buttonId = this.props.id ? `${this.props.id}__button` : this.uid;
+
+    const tileContentId = `tile-content-a11y-${this.instanceId}`;
+
     return (
       // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
-      <div
+      <aside
```

Comment:
Should these be an `aside` by default?

Reply:
@dakahn what do you think?

Reply:
Yeah, I think so. The `aside` element communicates our intended usage for this component (_I think_). Asides are right alongside `main content`, but also work as totally separate pieces. A role of complimentary means this will be treated as a landmark as well.

Reply:
There's is one point I came across during research. If you have multiple complimentary items on the page, they should have _unique_ labels. Something to keep in mind when demonstrating use or in our spec writing, maybe.

Reply:
THAT SAID! There was quite a bit of discussion offline about the implications of `aside` vs our user's theoretical use cases (where `aside` may not be appropriate). Rather than open that can of worms, maybe we should keep this a humble `div`?